src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj

```diff
@@ -19,6 +19,12 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Linux_Release|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Debug|AnyCPU' " />
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'OSX_Release|AnyCPU' " />
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FreeBSD_Debug|AnyCPU' "/>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'FreeBSD_Release|AnyCPU' "/>
+  <PropertyGroup>
+    <TargetsBSD Condition="'$(TargetsOSX)' == 'true' or '$(TargetsFreeBSD)' == 'true'">true</TargetsBSD>
+  </PropertyGroup>
```

Comment:
> That means FreeBSD, NetBSD, OpenBSD, DargonFly BSD etc. OS X is not a BSD, as we seem to agree.

I wasn't making any claims about whether OS X is or is not BSD. I personally don't see that as particularly important, at least not for this aspect of the project (I agree it's more interesting if you start exposing functionality that tells you whether something is or is not something, but even then it's almost always being used as a proxy to understand whether something does or does not support something). What matters to me is whether a particular API can be used on a particular OS and being able to describe the heritage of that API so as to more easily determine its potential applicability elsewhere and to help differentiate it when needed.

> to refer to an API originating from the BSD family

Yes, that's how the entire Interop folder and Targets* scheme is set up. If it makes you more comfortable, imagine that "API" is an implicit suffix on the name of all the Interop subfolders (including Windows) and on all of the Targets conditions, as that is what's meant. But as it's relevant everywhere, it doesn't need to be included anywhere.

> Rather put those files in the Unix folder and name them as *.BSDAPI.cs.

Taking that logic to the extreme, you could imagine a single Interop flat folder containing just files where every file ends in .WindowsAPI.cs, .UnixAPI.cs, .BSDAPI.cs, .OSXAPI.cs, .FreeBSDAPI.cs, etc. The whole point of the subfolders is to provide that same meaning but with more structure and less clutter.